lib/system.nim

```diff
@@ -443,6 +443,11 @@ type
   WriteIOEffect* = object of IOEffect  ## Effect describing a write IO operation.
   ExecIOEffect* = object of IOEffect   ## Effect describing an executing IO operation.
 
+when defined(genode):
+  type RpcEffect* = object of RootEffect
```

Comment:
I don't want genode specific effects in the core.

Reply:
Ok, I will put it with the libraries I will keep with the Genode toolchain.